css/index.css

```diff
@@ -1,3 +1,450 @@
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+header nav {
+  display: flex;
+  margin-top: 35px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+header nav a {
+  margin-right: 35px;
+  /* margin-left: 27px; */
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+.cover {
+  display: flex;
+  justify-content: space-between;
+}
+
+.cover .cover-content {
+  vertical-align: top;
+  font-size: 40px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 50px;
+  margin-bottom: -9px;
+  display: flex;
+  flex-direction: column;
+}
+@media (max-width: 500px) {
+  .cover .cover-content {
+    margin-top: 0;
+  }
+}
+
+.cover img {
+  display: flex;
+  margin-top: 69px;
+}
+
+.title {
+  margin-bottom: 5px;
+}
+
+button {
+  background: white;
+  /* vertical-align: top; */
+  margin-top: 1px;
+  padding: 3px 24px;
+  font-size: 12px;
+  border: 1px solid black;
+  border-radius: 5px;
+  cursor: pointer;
+  background: white;
+}
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 48px;
+  padding-top: 33px;
+  margin-bottom: -50px;
+  display: flex;
+}
+
+.top-content {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
+
+.main-content .top-content .text-content {
+  width: 44%;
+  vertical-align: top;
+  margin: 15px 25px;
+}
+
+.middle-img {
+  margin-top: 69px;
+  width: 100%;
+}
+
+.bottom {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
+
+.bottom-content .bottom .text-content {
+  display: flex;
+  width: 29%;
+  vertical-align: top;
+  margin: 15px 15px;
+}
+
+.bottom-content {
+  margin-top: 10px;
+  display: flex;
+}
+
+.bottom .text-content {
+  /* display: flex; */
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
+
+.contact {
+  margin-top: 0;
+  padding: 39px 10px 0;
+}
+
+.contact h4 {
+  margin-top: 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 80px 70px;
+}
+
+.copyright {
+  margin-left: 390px;
+  margin-bottom: 50px;
+}
+
+/* services styles */
+
+.top-header {
+  margin: 40px 0;
+}
+
+.labels {
+  font-size: 25px;
+}
+.squares {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.info-card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.squares .info-card {
+  border: 2px solid black;
+  background: lightgray;
+  justify-content: space-between;
+  width: 45%;
+  margin: 5px;
+  padding: 15px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.end {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.end .end-content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 45%;
+  margin: 5px;
+  padding: 10px;
+  margin-top: 15px;
+  margin-bottom: 25px;
+}
+
+.end .end-content p {
+  margin-bottom: 20px;
+}
+
+.end .end-content li {
+  margin-bottom: 15px;
+}
+
+@media (max-width: 800px) {
```

Comment:
Code is clean and readable. All in one place, but I do recommend if you do one thing, be consistent with it. If you want to put them all in one place, you gotta stick to it.